browser.js

```diff
@@ -190,11 +253,11 @@ Require.loadScript = function (location) {
 Require.ScriptLoader = function (config) {
     var hash = config.packageDescription.hash;
     return function (location, module) {
-        return Promise.fcall(function () {
+        return Promise.try(function () {
 
             // short-cut by predefinition
             if (definitions[hash] && definitions[hash][module.id]) {
-                return definitions[hash][module.id].promise;
+                return definitions[hash][module.id];
```

Comment:
This breaks preloading and is related to the Promise.resolve() replacing Promise.defer(). You’ll need to recreate defer().

```js
function defer() {
    var deferrred = {};
    deferred.promise = new Promise(resolve, reject) {
        deferred.resolve = resolve;
        deferred.reject = reject;
    });
    return deferred;
}
```